app/views/alchemy/admin/pages/_page.html.erb

```diff
@@ -1,4 +1,4 @@
-<li id="page_{{id}}" class="page_level_{{level}} {{page_layout}}" data-slug="{{slug}}" data-restricted="{{restricted}}" data-visible="{{visible}}" data-external="{{redirects_to_external}}">
+<li id="page_{{id}}" class="page_level_{{level}} {{page_layout}} {{#unless permissions.edit }}invalid_user_type_link {{/unless }}" data-slug="{{slug}}" data-restricted="{{restricted}}" data-visible="{{visible}}" data-external="{{redirects_to_external}}">
```

Comment:
Since we don't have styles for `.invalid_user_type_link` we can remove this here, or do you have custom styles you want to use?